packages/plugin-calendar/src/index.js

```diff
@@ -0,0 +1,74 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '@ciscospark/plugin-wdm';
+import '@ciscospark/plugin-encryption';
+import '@ciscospark/plugin-conversation';
+
+import {registerPlugin} from '@ciscospark/spark-core';
+import Calendar from './calendar';
+import config from './config';
+import {has} from 'lodash';
+
+registerPlugin(`calendar`, Calendar, {
+  config,
+  payloadTransformer: {
+    predicates: [
+      {
+        name: `decryptPropSubject`,
+        direction: `inbound`,
+        test(ctx, response) {
+          return Promise.resolve(has(response, `body.items[0].encryptedSubject`));
```

Comment:
currently, only GET is supported by the server. Will enhance this when the POST is supported.